layouts/system76/darp9/default.json

```diff
@@ -0,0 +1,425 @@
+{
+  "model": "system76/darp9",
+  "version": 1,
+  "map": {
+    "K00": [
+      "ESC",
+      "ESC"
+    ],
+    "K01": [
+      "F1",
+      "TOUCHPAD"
+    ],
+    "K02": [
+      "F2",
+      "MIC_MUTE"
+    ],
+    "K03": [
+      "F3",
+      "MUTE"
+    ],
+    "K04": [
+      "F4",
+      "F4"
+    ],
+    "K05": [
+      "F5",
+      "VOLUME_DOWN"
+    ],
+    "K06": [
+      "F6",
+      "VOLUME_UP"
+    ],
+    "K07": [
+      "F7",
+      "DISPLAY_MODE"
+    ],
+    "K08": [
+      "F8",
+      "BRIGHTNESS_DOWN"
+    ],
+    "K09": [
+      "F9",
+      "BRIGHTNESS_UP"
+    ],
+    "K0A": [
+      "F10",
+      "CAMERA_TOGGLE"
+    ],
+    "K0B": [
+      "F11",
+      "AIRPLANE_MODE"
+    ],
+    "K0C": [
+      "F12",
+      "SUSPEND"
+    ],
+    "K0D": [
+      "PRINT_SCREEN",
+      "PRINT_SCREEN"
+    ],
+    "K0E": [
+      "INSERT",
+      "INSERT"
+    ],
+    "K0F": [
+      "DEL",
+      "DEL"
+    ],
+    "K0G": [
+      "HOME",
+      "HOME"
+    ],
+    "K0H": [
+      "END",
+      "END"
+    ],
+    "K0I": [
+      "PGUP",
+      "PGUP"
+    ],
+    "K0J": [
+      "PGDN",
+      "PGDN"
+    ],
+    "K10": [
+      "TICK",
+      "PLAY_PAUSE"
+    ],
+    "K11": [
+      "1",
+      "FAN_TOGGLE"
+    ],
+    "K12": [
+      "2",
+      "2"
+    ],
+    "K13": [
+      "3",
+      "3"
+    ],
+    "K14": [
+      "4",
+      "4"
+    ],
+    "K15": [
+      "5",
+      "5"
+    ],
+    "K16": [
+      "6",
+      "6"
+    ],
+    "K17": [
+      "7",
+      "7"
+    ],
+    "K18": [
+      "8",
+      "8"
+    ],
+    "K19": [
+      "9",
+      "9"
+    ],
+    "K1A": [
+      "0",
+      "0"
+    ],
+    "K1B": [
+      "MINUS",
+      "MINUS"
+    ],
+    "K1C": [
+      "EQUALS",
+      "EQUALS"
+    ],
+    "K1D": [
+      "BKSP",
+      "BKSP"
+    ],
+    "K1E": [
+      "NUM_LOCK",
+      "NUM_LOCK"
+    ],
+    "K1F": [
+      "NUM_SLASH",
+      "KBD_STEP"
```

Comment:
Oh, actually, there's a test failing in CI because `KBD_STEP` isn't in `picker.json`.

Easily enough to add to the picker if it should be, but I don't see `KBD_STEP` anywhere in https://github.com/system76/ec?